### ISSUE TITLE
Accessibility bug: thumbnails not keyboard-activatable

### ISSUE THREAD

**Issue by scoutb-cogapp** [UniversalViewer/universalviewer#1068]
**UV version:** _universalviewer@4.0.25_

**I'm submitting a:** *bug report => please fork one of these [codesandbox examples](https://github.com/UniversalViewer/universalviewer/wiki/UV-Examples) with a repro of your issue and include a link to it below*

### Page area
thumbnails

### Issue description
Inside the contents sidebar (left), keyboard users cannot activate images from the thumbnails. They can tab through them but it is not possible to then activate the image to make it display in the image viewer (in the same way a mouse-click would).

### Steps to reproduce
1. open [this manifest](https://www.universalviewer.dev/#?xywh=583%2C1062%2C2492%2C1379&iiifManifestId=https%3A%2F%2Feap.bl.uk%2Farchive-file%2FEAP500-1-1%2Fmanifest&cv=)
2. Tab to any of the thumbnails in the left-hand sidebar (except the currently active one)
3. Try to activate the image to display in the main image viewer  

### Expected behaviour
pressing return key will switch the image in the main image viewer to the selected thumbnail's image

### WCAG criterion
2.1.1 Keyboard (Level A)

### Related code

```html
<div class="thumb oneCol" tabindex="0"><div class="wrap" style="height: 91px;"><img src="https://images.eap.bl.uk/EAP500/EAP500_1_1/2.jp2/full/90,/0/default.jpg" alt="2"></div><div class="info"><span class="label" title="2">2&nbsp;</span></div></div>
```

**Comment by danihagosbl** [comment]
@jamesmisson can you assign this issue to me then I will have a look at this issue.

**Comment by damodeburca** [comment]
@danihagosbl Hello, I have testedthis in vercel and the fix looks good, the user can tab forward and backwards and using the keyboard enter key can activate the selected image in the thumbnail view to render the same image in the main gallery. Nice work :)
